examples/bind/turing.amp.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
```

Comment:
How would you feel about adding a high-level explanation of what's supposed to happen in this demo in a comment.